larreco/RecoAlg/TCAlg/PFPUtils.cxx

```diff
@@ -1891,6 +1891,7 @@ namespace tca {
         auto& tp = slc.tjs[tp3d.TjID - 1].Pts[tp3d.TPIndex];
         sfTPs.push_back(tp);
         wire = std::nearbyint(tp.Pos[0]);
+        if (wire >= slc.nWires[pln]) break;
```

Comment:
Hi @brballer 
Would the geometry service be helpful here or would Thea function calls be too slow?  I think there are functions to find wires for a given position/plane: https://nusoft.fnal.gov/larsoft/doxsvn/html/classgeo_1_1GeometryCore.html#a7f9644be6b7a420fbeeb54c9ded33d62